index.js

```diff
@@ -1,12 +1,20 @@
 function handleVueDestruction(vue) {
-  var turbolinksEvent = vue.$options.turbolinksDestroyEvent || 'turbolinks:visit';
+  const turbolinksEvent = vue.$options.turbolinksDestroyEvent || 'turbo:visit';
+
   document.addEventListener(turbolinksEvent, function teardown() {
     vue.$destroy();
     document.removeEventListener(turbolinksEvent, teardown);
   });
+
+  if (!vue.$options.turbolinksDestroyEvent) {
```

Comment:
Probably not the best way to do it, but I would like to keep the support for both Turbolinks and Turbo

Reply:
We should be able to check if Turbo or Turbolinks is defined. 

That would let us set the `event` dynamically to `turbo:load` or `turbolinks:load` depending on what's being used so we can continue supporting both. Same for the destroy event.

Maybe just make a helper method for each?